md/sv2-workshop.md

```diff
@@ -200,7 +197,7 @@ server=1
 connect=75.119.150.111 # genesis node
 rpcuser=username
 rpcpassword=password
-sv2port=8442
+sv2port=38442
```

Comment:
we shouldn't have switched this

it needs to match [jds-config-sv2-workshop.toml](https://github.com/stratum-mining/stratum/blob/8f0d5b00fc7e611f99cfd2d4523b655d98a87109/roles/jd-server/jds-config-sv2-workshop.toml#L18)